# Patches for microb-engine
# Oleg Romashin <oleg.romashin@nokia.com>
# Anton Rogaynis <anton.rogaynis@nokia.com>
# Bug 359812: No reason to use XUL based button in netError.xhtml
# Bug 61877: Additional problem page icon not displayed in the Task navigator Application switcher  
Index: mozilla/docshell/resources/content/netError.xhtml
===================================================================
--- mozilla/docshell/resources/content/netError.xhtml.orig	2007-10-13 14:40:50.000000000 -0400
+++ mozilla/docshell/resources/content/netError.xhtml	2007-10-15 11:10:43.000000000 -0400
@@ -58,7 +58,7 @@
     <title>&loadError.label;</title>
     <link rel="stylesheet" href="chrome://global/skin/netError.css" type="text/css" media="all" />
     <!-- XXX this needs to be themeable -->
-    <link rel="icon" type="image/png" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABGdBTUEAANbY1E9YMgAAABl0RVh0U29mdHdhcmUAQWRvYmUgSW1hZ2VSZWFkeXHJZTwAAAICSURBVHjaYvz//z8DJQAggJhwiDPvnmlzc2aR0O+JGezt+AwACCCsBhxfYhn59N41FWtXIxZOLu70niRGXVwGAAQQNgNYHj96O8HaWYdJW5ubwd4/mI2Ng7sblwEAAYRhwMm1URk/vn4SUNWVYGD8+YZBXZOZm5OLzRjoCmNsBgAEEKoBN82Y7l851GLrqMjM8Oc7A8O/3wwMP54wuAQFCXNycUzGZgBAAKEYcOaKZO2/f//5FbUVgBrfMoRVcgHpNwyKGjKMXDwCan0prFboBgAEELIBzDcvXyy2cVZhYPj9GWj7H4jo/38MDJ9OMDj7O/KzsjH3oxsAEEBwA/bNNipiZf7FI6cqwcDw8x2qqp8fGORUpVn4BEXlgGHhhCwFEEAwA9gfP3hdZ+Oizcjw+wvCdjgAuuLrFQbXIH9hTm7uqcgyAAEENuD4ctcebm5mbikFYRTbV7V/Q6j88Z5BSuY7q4CQgAjQFR4wYYAAAhtw89L5ZFsnRaDtn4CW/YXrAQcisit+PGVwDgrnZ2NnnwATBQggpsNLvGYLCAmxi8tLARWg+h3FBVBXSEj/ZZWQkRcCuiIQJAQQQCyvnj5KMDTkZ2JgYmRg4FchnHv+vmEwttLmeXT3VjKQtx4ggFgk5TXebV63UfT3ijOMxOZAVlZWdiB1EMQGCCBGSrMzQIABAFR3kRM3KggZAAAAAElFTkSuQmCC" />
+    <link rel="icon" type="image/png" href="file:///usr/share/icons/hicolor/26x26/hildon/qgn_note_gene_syswarning.png" />
 
     <script type="application/x-javascript"><![CDATA[
       // Error url MUST be formatted like this:
--- mozilla.orig/docshell/resources/content/netError.xhtml
+++ mozilla/docshell/resources/content/netError.xhtml
@@ -205,18 +205,17 @@
         </div>
 
         <!-- Long Description (Note: See netError.dtd for used XHTML tags) -->
         <div id="errorLongDesc" />
 
       </div>
 
       <!-- Retry Button -->
-      <xul:button xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
-                  id="errorTryAgain" label="&retry.label;" oncommand="retryThis(this);" />
+      <button id="errorTryAgain" onclick="retryThis();">&retry.label;</button>
 
     </div>
 
     <!--
     - Note: It is important to run the script this way, instead of using
     - an onload handler. This is because error pages are loaded as
     - LOAD_BACKGROUND, which means that onload handlers will not be executed.
     -->
